pkg/apis/elasticsearch/v1alpha1/types.go

```diff
@@ -136,3 +136,45 @@ const (
 	// Unmanaged means that the operator will not take any action related to the component
 	ManagementStateUnmanaged ManagementState = "Unmanaged"
 )
+
+// ClusterCondition contains details for the current condition of this elasticsearch cluster.
+type ClusterCondition struct {
+	// Type is the type of the condition.
+	Type ClusterConditionType `json:"type"`
+	// Status is the status of the condition.
+	Status ConditionStatus `json:"status"`
+	// Last time the condition transitioned from one status to another.
+	LastTransitionTime metav1.Time `json:"lastTransitionTime"`
+	// Unique, one-word, CamelCase reason for the condition's last transition.
+	Reason string `json:"reason,omitempty" protobuf:"bytes,5,opt,name=reason"`
+	// Human-readable message indicating details about last transition.
+	Message string `json:"message,omitempty" protobuf:"bytes,6,opt,name=message"`
+}
+
+// ClusterConditionType is a valid value for ClusterCondition.Type
+type ClusterConditionType string
+
+// These are valid conditions for elasticsearch node
+const (
+	ClusterSettingsUpdateNeeded ClusterConditionType = "SettingsUpdateNeeded"
```

Comment:
This is for the operator to use to know what it should be doing. This is different than `scaling up`, which has a definition below.

Reply:
@jcantrill we dynamically change cluster settings, in this example node quorum, based on the number of nodes in the cluster

Reply:
My point is I think it reads "I need you to do something" instead of saying "I am doing soming".  I believe its a subtle choice in wording that should be corrected

Reply:
I'd suggest `UpdatingSettings` then

Reply:
changed to UpdatingSettings